WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -178,10 +178,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 {
     DDLogVerbose(@"didFinishLaunchingWithOptions state: %d", application.applicationState);
 
-    // Launched by tapping a notification
-    if (application.applicationState == UIApplicationStateActive) {
```

Comment:
We probably should continue checking if the app is in an active and not a background state when handing remote notifications.